### ISSUE TITLE
webvr-polyfill breaks building AFrame with Node/ember-cli

### ISSUE THREAD

**Issue by gabrielgrant** [immersive-web/webvr-polyfill#322]
##### Description:

I'm using webvr-polyfill (installed via a dependency on AFrame) in a project compiled with ember-cli. The project fails to build due with the following traceback:

```
ReferenceError: screen is not defined
    at /home/gabriel/repos/ff/node_modules/aframe/node_modules/webvr-polyfill/build/webvr-polyfill.js:2315:1
    at classCallCheck (/home/gabriel/repos/ff/node_modules/aframe/node_modules/webvr-polyfill/build/webvr-polyfill.js:227:19)
    at /home/gabriel/repos/ff/node_modules/aframe/node_modules/webvr-polyfill/build/webvr-polyfill.js:228:2
    at createCommonjsModule (/home/gabriel/repos/ff/node_modules/aframe/node_modules/webvr-polyfill/build/webvr-polyfill.js:115:35)
    at /home/gabriel/repos/ff/node_modules/aframe/node_modules/webvr-polyfill/build/webvr-polyfill.js:147:26
    at commonjsGlobal (/home/gabriel/repos/ff/node_modules/aframe/node_modules/webvr-polyfill/build/webvr-polyfill.js:101:82)
    at Object.<anonymous> (/home/gabriel/repos/ff/node_modules/aframe/node_modules/webvr-polyfill/build/webvr-polyfill.js:104:2)
    at Module._compile (module.js:573:30)
    at Object.Module._extensions..js (module.js:584:10)
    at Module.load (module.js:507:32)
```

This was working previously. The breakage was caused by upgrading AFrame from v0.7.1 to 0.8.2, which bumped webvr-polyfill from v0.9.36 to v0.10.5

##### Additional Information:

* webvr-polyfill version: 0.10.5
* Browser name/version/release channel: Build environment is node v8.4.0
* Operating System: Ubuntu

**Comment by gabrielgrant** [comment]
The problematic line seems to have been introduced here: https://github.com/immersive-web/webvr-polyfill/commit/a5d75ee7eb3491935d79da8bc85d0a807d4fd829#diff-c9f2bc69ace9478cfd1156aabd2a8e31R2300

**Comment by kellyselden** [comment]
My previous PRs to get node working may offer you some guidance on how to fix. https://github.com/immersive-web/webvr-polyfill/pulls?q=is%3Apr+is%3Aclosed+author%3Akellyselden

**Comment by jsantell** [comment]
The latest patch for adding `referenceFrame` to the RelativeOrientationSensor removes the screen.orientation code that will fix this: https://github.com/immersive-web/cardboard-vr-display/pull/28

**Comment by jsantell** [comment]
Thanks for your patience! The fix has been pulled in from cardboard-vr-display in 41421fd66894689050e291a7878b51acd03c1080 and released as `0.10.7`
